ports/libspnav/portfile.cmake

```
@@ -0,0 +1,50 @@
+vcpkg_fail_port_install(ON_TARGET "windows" "uwp" "osx")
+
+vcpkg_from_github(
+	OUT_SOURCE_PATH SOURCE_PATH
+	REPO FreeSpacenav/libspnav
+    REF v0.2.3
+    SHA512 fe01a8e8496aa5648553ad8a769aa15a4de127e3cf0a1d2c56353bf9757fe025383d64a07709c33199f658aa00adcb1fa754891f7de2250610a56664bc107c45
+    HEAD_REF master
```

Mixed whitespace styles.
```suggestion
	OUT_SOURCE_PATH SOURCE_PATH
	REPO FreeSpacenav/libspnav
	REF v0.2.3
	SHA512 fe01a8e8496aa5648553ad8a769aa15a4de127e3cf0a1d2c56353bf9757fe025383d64a07709c33199f658aa00adcb1fa754891f7de2250610a56664bc107c45
	HEAD_REF master
```

---

The whitespace should be consistent and it should be four spaces.